.circleci/config.yml

```diff
@@ -392,7 +392,7 @@ jobs:
           valgrind_config: << parameters.valgrind_config >>
       - run:
           name: Run extension tests with leak detection
-          command: make << parameters.unit_test_target >> BUILD_DIR=tmp/build_extension JUNIT_RESULTS_DIR=$(pwd)/test-results
+          command: make << parameters.unit_test_target >> BUILD_DIR=$(pwd)/tmp/build_extension JUNIT_RESULTS_DIR=$(pwd)/test-results
```

Comment:
Any insight here as to why `pwd` was added? I occasionally build it somewhere else, like in `/tmp/build` (note leading slash).

Reply:
Yes, `TEST_PHP_SRCDIR` needs to be an absolute path. You can still build it somewhere else as long as `BUILD_DIR` is an absolute path.